src/components/accordion/accordion.js

```diff
@@ -0,0 +1,245 @@
+
+/*
+  Accordion
+
+  This allows a collection of sections to be collapsed by default,
+  showing only their headers. Sections can be exanded or collapsed
+  individually by clicking their headers. An "Open all" button is
+  also added to the top of the accordion, which switches to "Close all"
+  when all the sections are expanded.
+
+  The state of each section is saved to the DOM via the `aria-expanded`
+  attribute, which also provides accessibility.
+
+*/
+
+import { nodeListForEach } from '../../common'
+import '../../vendor/polyfills/Function/prototype/bind'
+import '../../vendor/polyfills/Element/prototype/classList'
+
+function Accordion ($module) {
+  this.$module = $module
+  this.$sections = $module.querySelectorAll('.govuk-accordion__section')
+  this.$openAllButton = ''
+}
+
+Accordion.prototype.init = function () {
+  // Check module exists
+  var $module = this.$module
+  if (!$module) {
+    return
+  }
+
+  this.moduleId = $module.getAttribute('id')
+
+  nodeListForEach(this.$sections, function ($section, i) {
+    // Set header attributes
+    var header = $section.querySelector('.govuk-accordion__section-header')
+    this.setHeaderAttributes(header, i)
+
+    this.setExpanded(this.isExpanded($section), $section)
+
+    // Handle events
+    header.addEventListener('click', this.onToggleExpanded.bind(this, $section))
+
+    header.addEventListener('click', function() {
+      console.log('clicked')
```

Comment:
We should remove all `console.log`'s as IE8 throws the error

```
 'console' is undefined
```
(unless Dev tools are open)

Reply:
@hannalaakso opps, well spotted, I should have removed those debugging lines earlier.